gtda/diagrams/features.py

```diff
@@ -32,6 +32,10 @@ class PersistenceEntropy(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
+    normalize_nb_points : bool, optional, default: ``False``
+        Normalize PersistenceEntropy by the log of the total persistence of
```

Comment:
`PersistenceEntropy` is the name of a transformer and shouldn't be used here. Can I suggest something along these lines:
```
When ``True``, the persistence entropy of each diagram is normalized by the logarithm of the sum of lifetimes of all points in the diagram. Can aid comparison between diagrams in an input collection when these have different numbers of (non-trivial) points.
```

Reply:
that's better ,thanks!